discord/interactions.py

```diff
@@ -303,10 +303,28 @@ async def original_message(self) -> InteractionMessage:
         )
         state = _InteractionMessageState(self, self._state)
         message = InteractionMessage(state=state, channel=channel, data=data)  # type: ignore
-        self._original_message = message
+        self._original_response = message
         return message
 
-    async def edit_original_message(
+    async def original_message(self, **kwargs):
+        """An alias for :meth:`original_response`.
+        .. versionadded:: 2.0
```

Comment:
Incorrect, this would probably be added in 2.2 at most
Also you need a newline in between the versionadded statement and the description.

Reply:
I have removed it.